staging/src/k8s.io/legacy-cloud-providers/azure/azure_loadbalancer_test.go

```diff
@@ -378,6 +379,12 @@ func TestEnsureLoadBalancerDeleted(t *testing.T) {
 			service:           getResourceGroupTestService("test4", "random-rg", "1.2.3.4", 80),
 			expectCreateError: true,
 		},
+		{
```

Comment:
@phiphi282 do you know why this test case failed here?
https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/94714/pull-kubernetes-bazel-test/1304356108487364610
```
--- FAIL: TestEnsureLoadBalancerDeleted (0.02s)
    azure_loadbalancer_test.go:403: 
        	Error Trace:	azure_loadbalancer_test.go:403
        	Error:      	Expected value not to be nil.
        	Test:       	TestEnsureLoadBalancerDeleted
        	Messages:   	TestCase[4]: annotated service with different resourceGroup shouldn't be created but should be deleted successfully
    azure_loadbalancer_test.go:421: 
        	Error Trace:	azure_loadbalancer_test.go:421
        	Error:      	Not equal: 
        	            	expected: 1
        	            	actual  : 0
        	Test:       	TestEnsureLoadBalancerDeleted
        	Messages:   	TestCase[4]: annotated service with different resourceGroup shouldn't be created but should be deleted successfully
```

Reply:
I haven't understood completely why, but for v1.18 if using a different resource group without specifying an ip, a loadbalancer will actually be created.
To fix this we could just add an ip to the testcase.